tests/test_zeyple.py

```diff
@@ -106,10 +141,32 @@ def test_process_message_with_multipart_message(self):
             Yy90ZXN0JyB3d3ctZGF0YQo=
 
             --=_504b4162.Gyt30puFsMOHWjpCATT1XRbWoYI1iR/sT4UX78zEEMJbxu+h--
-        """), ["torvalds@linux-foundation.org"])
+        """), [TEST1_EMAIL])
 
         assert emails[0]['X-Zeyple'] is not None
-        assert emails[0].is_multipart()
-        for part in emails[0].walk():
-            assert not is_encrypted(part.as_string().encode('utf-8'))
+        assert not emails[0].is_multipart()  # GPG encrypt the multipart
+        assert is_encrypted(emails[0].get_payload().encode('utf-8'))
+
+    def test_process_message_with_multiple_recipients(self):
+        """Encrypt a message with multiple recipients"""
+
+        content = "Content"
+
+        emails = self.zeyple.process_message(dedent("""\
+            Received: by example.org (Postfix, from userid 0)
+                id DD3B67981178; Thu,  6 Sep 2012 23:35:37 +0000 (UTC)
+            To: """ + ', '.join([TEST1_EMAIL, TEST2_EMAIL]) + """
+            Subject: もしもし with Unicode
+            Message-Id: <20120906233537.DD3B67981178@example.org>
+            Date: Thu,  6 Sep 2012 23:35:37 +0000 (UTC)
+            From: root@example.org (root)
+
+            """ + content + """
+        """), [TEST1_EMAIL, TEST2_EMAIL])
+
+        assert len(emails) == 2  # It had two recipients
 
+        for m in emails:
+            assert m['X-Zeyple'] is not None
+            payload = self.decrypt(m.get_payload()).strip()
+            assert payload == six.b(content)
```

Comment:
I think it'd be great to replace `assert is_encrypted(...)` with `assert self.decrypt(...) == plain_text_content` for all tests now that we have a way to decrypt messages. `is_encrypted()` was to make sure the email was actually encrypted but if we could both check that it is encrypted and properly decryptable, it's even better. Thoughts?